modules/ply/src/ply-streaming-loader.js

```diff
@@ -0,0 +1,14 @@
+// description: 'PLY - Polygon File Format (aka Stanford Triangle Format)'
```

Comment:
- Nit: Just call it `PLYStreamLoader`?
- Any reason we cannot parse as the file is streamed by default?

Reply:
> Any reason we cannot parse as the file is streamed by default?

Make the streaming loader the default loader would be the long term goal, the biggest reason that won't work right now is that the streaming loader only supports the ASCII version, and we will also need some more smarts in `loadFile` to open streams in some cases instead of just reading the file.

> Nit: Just call it `PLYStreamLoader`?

Sure. And I'll put an underscore on it for now. As you suggested, this loader should go away eventually.